Documentation/ApiOverview/JavaScript/Forms/Index.rst

```diff
@@ -0,0 +1,33 @@
+.. include:: ../../../Includes.txt
+
+
+.. _javascript-form-helpers:
+
+=======================
+JavaScript Form Helpers
+=======================
+
+Empty Checkbox Handling
+=======================
+
+.. code-block:: html
+
+   <input type="checkbox" name="setting" value="1" data-empty-value="0"
+      data-global-event="change" data-action-navigate="$data=~s/$value/">
+
+Checkboxes used to send a particular value when being unchecked can be achieved by using
+:html:`data-empty-value="0"` - in case this attribute is omitted, an empty string `''` is sent.
```

Comment:
```suggestion
Checkboxes used to send a particular value when unchecked can be achieved by using
:html:`data-empty-value="0"`. If this attribute is omitted, an empty string `''` is sent.
```
OR
To send a particular value when a checkbox is unchecked, use :html:`data-empty-value="0"`. When this attribute is omitted, an empty string `''` is sent.